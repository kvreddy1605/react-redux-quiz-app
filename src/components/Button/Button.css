Button{
    background-color: #955251;
    color: white;
    border: none;
    padding: 10px 20px;
    border-radius: 5px;
    font-size: 15px;
}