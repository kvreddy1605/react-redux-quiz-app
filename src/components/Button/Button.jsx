import React from 'react';
import classes from './Button.css';

const button = (props) => {
    return(
        <button className={classes.Button} onClick={props.clicked}>{props.children}</button>
    )
}

export default button;