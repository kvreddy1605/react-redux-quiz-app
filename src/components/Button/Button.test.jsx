import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../Button/Button'

describe('Button Component Test', () => {
    const props = {
        clicked: jest.fn(),
        children: 'This is Button Children'
    };
    const Component = <Button {...props}/>
    it('should match latest snapshot', ()=>{
        const snapshot = renderer.create(Component)
        expect(snapshot.toJSON()).toMatchSnapshot()
    })
})