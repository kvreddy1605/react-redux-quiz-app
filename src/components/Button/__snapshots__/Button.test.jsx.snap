// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Test should match latest snapshot 1`] = `
<button
  className={undefined}
  onClick={[Function]}
>
  This is Button Children
</button>
`;
